### PR TITLE
Metastation fixes, adds nodrop subtype for superhero costumes v2

### LINKED ISSUE #5019: NODROP Owlman Costume Available in Metastation
**Problem Description**:
See: Title

**What did you expect to happen**: 
For either there not to be a NODROP item in maint that doesn't have any clue it would be NODROP.

**What happened instead**:
There is.

**Why is this bad/What are the consequences:**
If a player picks this up without knowing it is NODROP, they're stuck with it.

**Steps to reproduce the problem**:
Not entirely certain where all its locations are, or where if it can differ, but one was found in maint to the right of toxins mixing.

@tigercat2000

### LINKED ISSUE #5025: Metastation hydroponics lacks gene machines
**Problem Description**:
No machines for genes extraction/application in hydroponics

**What did you expect to happen**: 
Machines to be there.

**What happened instead**:
Machines aren't there.

**Why is this bad/What are the consequences:**
Botanists can't  play with genes.

**Steps to reproduce the problem**:
Check hydroponics on metastation.

### LINKED ISSUE #5027: Metastation - Toxin Test Chamber Cameras
It seems as if the screens aren't connected to the cameras at the testing range.

You click on them but nothing happens. Not even the camera screen shows up.

### PR DESCRIPTION
**Pull request by LittleBigKid2000**
Holy hell this is a lot of fixes

(Unless I say boxstation/cyberiad, assume map changes are for metastation, so I don't have to type or paste metastation every time)
- Adds a NODROP subtype of the owl and griffin costumes to fix #5019. Superheroes are given the nodrop version.
- Adds plasma and reinforced plasma window spawners and replaces the one-directional plasma windows on a grille on Boxstation with them.
- Connects the BS and NT rep office's disposal bins to the disposal network, fixing #5020

- Adds two slime management consoles to xenobiology and adds a small beaker of sulphuric acid and a cyborg analyzer to robotics, fixing two parts of #5021

- Adds a full set of surgery tools, a space cleaner bottle, an anesthetic tank, a medical mask, and boxes of latex gloves, sterile masks, and body bags to robotics. Also adds a light fixture above the surgery table.

- Adds a light fixture to the little area before the secure lab and research shuttle dock

- Replaces the tinted windows to R&D and Robotics with reinforced windows, adds/fixes privacy shutters to replace some of the tinted windows. (Tested)
- Fixes the toxins mixing chamber airlock #5023 (Tested)
- Replaces the air pumps to and from the toxins mixing chamber with high capacity air pumps, fixing #5031.

- Adds every kind of monkey cube box to genetics, fixing #5024. Also puts the monkey cubes on a table where the wardrobe was and moves the wardrobe.

- Adds botany genetics machines to hydroponics. Fixes #5025.

- Adds lights to the escape pods, fixing #5026.
- Fixes #5027 by adding the 'research' and 'ss13' networks to the toxins range camera. (Tested)
- Fixes the names of the ERT and DS spawns, fixing #5029 and the duplicate #5030
- Also adds a bathroom and an office for ERT leaders to the ERT area thing.

- Adds a toolbox and pod lock buster on a table to the armory, fixing #5040.

- Adds spacepod keys to the HoS's office and security pod bay.
- Fixes the orientation of the shutters on the supply bridge, warehouse, and gateway.
- Adds a camera to the teleporter room on the minisat, fixing  #5028. Also removes a vent that wasn't connected to anything that was under the teleporter console.

### CODE REVIEW

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -5449,9 +5449,9 @@
...
-"caR" = (/obj/structure/grille,/obj/structure/window/plasmareinforced{dir = 8},/obj/structure/window/plasmareinforced,/obj/structure/window/plasmareinforced{dir = 1},/obj/machinery/atmospherics/pipe/simple/insulated,/obj/machinery/meter,/turf/simulated/floor/plating,/area/toxins/mixing)
-"caS" = (/obj/structure/grille,/obj/structure/window/plasmareinforced,/obj/structure/window/plasmareinforced{dir = 1},/turf/simulated/floor/plating,/area/toxins/mixing)
-"caT" = (/obj/structure/grille,/obj/structure/window/plasmareinforced{dir = 4},/obj/structure/window/plasmareinforced,/obj/structure/window/plasmareinforced{dir = 1},/obj/machinery/atmospherics/pipe/simple/insulated,/obj/machinery/meter,/turf/simulated/floor/plating,/area/toxins/mixing)
+"caR" = (/turf/simulated/floor/plasteel{dir = 4; icon_state = "whitebluecorner"; tag = "icon-whitebluecorner"},/area/security/medbay)
+"caS" = (/obj/machinery/sleeper{dir = 4},/turf/simulated/shuttle/floor{icon_state = "floor3"},/area/shuttle/escape)
+"caT" = (/obj/machinery/sleeper/upgraded{dir = 4},/turf/simulated/shuttle/floor{icon_state = "floor3"},/area/shuttle/administration)
```
Comment (Fox-McCloud) [inline _maps/map_files/cyberiad/cyberiad.dmm]: uhhhhh.....what did you change here

Comment (LittleBigKid2000) [inline _maps/map_files/cyberiad/cyberiad.dmm]: Huh, weird. I haven't messed with any of those areas.

**Comment by Fox-McCloud** [comment]
>Disclaimer: I haven't actually tested any of this.

Bad

**Comment by LittleBigKid2000** [comment]
The toxins mixing chamber has an airlock now.
Mapmerge thing fixed.
Coated r_walls replaced with regular r_walls.
Added a nutrimax back to hydroponics. Yes, there was, and now, more than one nutrimax.
I tested the toxins range camera, it works for me after the change.

**Comment by Fox-McCloud** [comment]
merge conflictsss.

Also, those wires need to be the same color as existing wires.

**Comment by monster860** [comment]
@Fox-McCloud This is *very much* mapmerged. Don't give the *not mapmerged* bullshit!

**Comment by LittleBigKid2000** [comment]
*SCREAM
FOX HOW DO I FIX MERGE CONFLICTS I STILL DON'T KNOW HOW

**Comment by monster860** [comment]
@LittleBigKid2000 Get bay mapmerge

**Comment by monster860** [comment]
@LittleBigKid2000 You know what? I'm gonna make a handy zip file for bay mapmerge.

**Comment by monster860** [comment]
@LittleBigKid2000 Handy zip file for bay mapmerge: http://puu.sh/q8NOn/1b957de0f8.zip

**Pull request closed without merging**